russstation/code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -27,4 +27,16 @@
 		M.fakevomit(0)
 	return ..()
 
+
+/datum/reagent/consumable/superlaughter/traitor_pen_variant
+	name = "Super Laughter"
+	description = "Funny until you're the one laughing."
+	metabolization_rate = 1.5 * REAGENTS_METABOLISM
+	color = "#FF4DD2"
+	taste_description = "laughter"
+
+/datum/reagent/consumable/superlaughter/on_mob_life(mob/living/carbon/M)
+	M.emote("laugh")
```

Comment:
I worry a series of regularly spaced laughs occurring every tick would both be obvious as to what happened to the person and a bit janky. Would it make sense to put this behind a prob check?

Reply:
Yea, I put a prob check to add some unevenness